### ISSUE TITLE
[Feature request] Add Android Wear projects to gdx-setup.jar

### ISSUE THREAD

**Issue by ammobyte** [libgdx/libgdx#2894]
Generating project from gdx-setup.jar would provide build for wear alongside "android", "desktop", "ios", "html", etc.

**Comment by MobiDevelop** [comment]
I don't think we'll be adding Android Wear to the setup. It should be relatively easy to set up Android Wear in your project after generating it.

**Comment by ammobyte** [comment]
Do you know about setting up Wear? I've been using LibGDX with android for a while, and I can't set up a project to work with Wear without quite a few errors, which makes me think having an option to pre-generate it would be helpful.

**Comment by MobiDevelop** [comment]
Nope, I've never tried it... Wear isn't really a great fit for libgdx.

What type of errors are you seeing?

**Comment by ammobyte** [comment]
It's been a bit since I've tried building a Wear project, so I'll have to reload some and see what I get, but I remember issues with Gradle that weren't solved by re-installing or re-downloading. I also tried using the source for flopsy droid, but there's several errors:
* ImageReference class import failed (learned that this is deprecated)
* GridPagerAdapter class import failed (don't know about this class)
* "Could not find com.android.support:support-v11:". Error log recommends installing Android Support Repository through the SDK manager, although it is installed and on revision 11 through the link to the SDK manager the error log provides.
